pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -109,13 +109,29 @@ var (
 			Buckets: []float64{30, 60, 120, 300, 600, 1200, 1800},
 		},
 	)
+	imagesetJobsDurationHistogram = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Name:    "imageset_jobs_duration_seconds",
```

Comment:
All our metrics should be prefixed with hive_. Could you update this and the ones above that aren't?